appveyor.yml

```diff
@@ -9,41 +9,9 @@ environment:
   matrix:
     # Note: 'default' toolchain is the same as 'vs-14-2015'
 
-    - TOOLCHAIN: "ninja-vs-12-2013-win64"
+    - TOOLCHAIN: "dummy"
       PROJECT_DIR: examples\bison
 
-    - TOOLCHAIN: "nmake-vs-12-2013-win64"
```

Comment:
Keep it:
* https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#excluding-tests